packages/core/src/domain/report/reportObservable.ts

```diff
@@ -0,0 +1,119 @@
+import { mergeObservables, Observable } from '../../tools/observable'
+import { DOM_EVENT, includes, addEventListener } from '../../tools/utils'
+import { monitor } from '../internalMonitoring'
+import type {
+  Report,
+  BrowserWindow,
+  ReportType,
+  ReportingObserverCallback,
+  ReportingObserverOption,
+  ReportingObserver as ReportingObserverInterface,
+} from './browser.types'
+
+declare const ReportingObserver: {
+  new (callback: ReportingObserverCallback, option: ReportingObserverOption): ReportingObserverInterface
+}
+
+export const CustomReportType = {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I wonder, why is it called `CustomReportType` and not simply `ReportType`?

Reply:
Same comment as https://github.com/DataDog/browser-sdk/pull/1332#discussion_r816747785